package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "@UrAvgDeveloper/governance-contracts",
+  "description": "",
+  "version": "1.0.0",
```

Comment:
I would start with 0.0.1, so when semantic-release is executed the version 1.0.0 will be automatically generated